akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingSettings.scala

```diff
@@ -83,7 +84,9 @@ object ClusterShardingSettings {
     val leastShardAllocationRebalanceThreshold:       Int,
     val leastShardAllocationMaxSimultaneousRebalance: Int,
     val waitingForStateTimeout:                       FiniteDuration,
-    val updatingStateTimeout:                         FiniteDuration)
+    val updatingStateTimeout:                         FiniteDuration,
+    val entityRecoveryRateInterval:                   FiniteDuration)
```

Comment:
I think there is a bit more work needed here to make it binary backwards compatible, the old constructor must be added